feature(splitbutton): 15706 add ability to have rotation of chevron icon

tri-harmoney opened the pull request:
Resolves https://github.com/primefaces/primeng/issues/15706

SoyDiego reviewed (commented):
I dont understand why is necessary the functionality rotate in the splitbutton 😕

tri-harmoney commented:
@SoyDiego I changed the proposal. We want to be able to rotate the chevron based on if the split button is expanded or not.
While not doing too much I added a css class `is-expanded`. This is a general solution and we can now rotate the chevron in our css based on that class. Do you agree more with this solution?

cetincakiroglu commented:
Hi,

Rotating the chevron is not in our design system. You can use `dropdownicon` template to customize the icon. Thanks a lot for the effort, closing the PR.
